pages/getting-started/writing-your-first-test.md

```diff
@@ -0,0 +1,92 @@
+
+> Note que o Cypress exibe a mensagem que não pode encontrar nenhum teste. Isso é normal - testes ainda não foram escritos! Você verá essa mensagem também caso exista algum erro ao analisar os arquivos de teste. Você pode abrir o Dev Tools para inspecionar o console e buscar por erros de sintaxe que impediu o Cypress de ler seus arquivos de testes.
+
+# Escreva seu primeiro teste
+
+Chegou a hora de escrever o nosso primeiro teste. Vamos então:
+
+1. Escrever nosso primeiro teste com sucesso.
+2. Escrever nosso primeiro teste com falha.
+3. Ver o Cypress recarregar o teste em tempo real.
+
+A medida que salvamos nosso novo teste, veremos que o navegador recarrega automaticamente em tempo real.
+
+Abra sua IDE favorita e adicione o código abaixo no arquivo de teste `sample_spec.js`.
+
+```
+describe('My First Test', () => {
+  it('Does not do much!', () => {
+    expect(true).to.equal(true)
+  })
+})
+```
+
+Quando salvar esse arquivo você deve ver o navegador recarregar.
+
+Embora não faça nada, esse é nosso primeiro teste com sucesso!
+
+No [log de comandos](https://docs.cypress.io/guides/core-concepts/test-runner.html#Command-Log) você vai ver que o Cypress exibe a suíte de teste, o teste e a primeira asserção (que deve estar em verde - sucesso).
+
+![Primeiro teste com sucesso](https://docs.cypress.io/img/guides/first-test.88031830.png)
+
+> Observe que o Cypress exibe uma mensagem sobre essa ser a página padrão [ao lado direito](https://docs.cypress.io/guides/core-concepts/test-runner.html#Application-Under-Test). Cypress assume que você irá sair e [visitar](https://docs.cypress.io/api/commands/visit.html) outra página na internet - mas também pode funcionar bem sem isso.
+
+Agora vamos escrever nosso primeiro teste com falha.
+
+```
+describe('My First Test', () => {
+  it('Does not do much!', () => {
+    expect(true).to.equal(false)
+  })
+})
+```
+
+Quando você salvar, poderá observar que o Cypress exibe o teste com falha em vermelho, uma vez que `true` não é igual a `false`.
+
+Cypress também exibe o log do erro e o pedaço do código onde a asserção falhou (quando disponível). Você pode clicar no link do arquivo em azul para abrir o arquivo em que o erro ocorreu [em sua IDE preferida](https://docs.cypress.io/guides/tooling/IDE-integration.html#File-Opener-Preference). Para ler mais sobre exibição de erros, leia sobre [Debugando Erros](https://docs.cypress.io/guides/guides/debugging.html#Errors).
+
+![Teste com falha](https://docs.cypress.io/img/guides/failing-test.971461e3.png)
+
+Cypress fornece um ótimo [Test Runner](https://docs.cypress.io/guides/core-concepts/test-runner.html) que exibe uma estrutura visual das suítes, testes e asserções. Em breve você verá também comandos, eventos, requisições de rede e mais.
+
+[Vídeo do primeiro teste com falha](https://docs.cypress.io/img/snippets/first-test-30fps.mp4)
```

Comment:
same question that I already asked above here

Reply:
This video doesn't have audio.

Reply:
great! we can add it 🚀